src/internet_identity/src/assets.rs

```diff
@@ -134,3 +142,57 @@ fn get_assets() -> [(&'static str, &'static [u8], ContentEncoding, ContentType);
         ),
     ]
 }
+
+pub fn increase_asset_counter(asset: String, status_code: u16) {
+    const MAX_TRACKED_ASSETS: u8 = 30;
+
+    if (200..400).contains(&status_code) {
+        // prevent clients from recording successful status codes for non-existing assets
+        trap_if_not_asset(&asset);
+
+        // prevent clients from recording impossible status codes for existing assets
```

Comment:
why not record "impossible" status codes? wouldn't those be the most interesting ones? also with this code in place I'm 100% sure we'll forget to update it if we change the return code for the faq

Reply:
No intermediary network component should modify success status codes, i.e. reporting status code 202 on an existing asset is a malicious input, so I discard it here.